global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "3.1.404",
+        "version": "5.0.201",
```

Comment:
This should be undone; not everyone will have access to .NET SDK 5.0, and no part of Chummer really needs it.

Reply:
Sorry. I didn't know how to create the environment, so I left the settings for me.
This change will be undone.